lib/elastic_apm/config.rb

```diff
@@ -182,8 +182,6 @@ def app=(app)
       case app_type?(app)
       when :sinatra
         set_sinatra(app)
-      when :rails
-        set_rails(app)
```

Comment:
@mikker can you think of a reason why we wouldn't want to remove these Rails checks in the Config?

Reply:
Could we remove the Sinatra ones too then, if we want to move it to the _starters_?

Reply:
Yes, let's do that 👍🏼